firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/ApkUpdater.java

```diff
@@ -84,6 +84,7 @@ public ApkUpdater(
     this.appDistributionNotificationsManager = appDistributionNotificationsManager;
     this.httpsUrlConnectionFactory = httpsUrlConnectionFactory;
     this.lifeCycleNotifier = lifeCycleNotifier;
+    cachedUpdateTask = new UpdateTaskImpl();
```

Comment:
I am unsure why we were not initializing our update tasks in the constructor, but I added it here since it was needed for a test.